.golangci.yml

```diff
@@ -32,7 +32,7 @@ linters:
     - errcheck
     - gocritic
     - gofmt
-    - goimports
+    - gci
```

Comment:
Good point! I have a PR up for that here https://github.com/cloudquery/.github/pull/107.
I'm not merging it yet as it will create a ton of noise in our repos. Planning to gradually update the repos, and then merge the sync PR.

I can also just do the `goimports -> gci` change separately, but again I'll have to fix any issues it will report.